probcal/data_modules/aaf_datamodule.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+import lightning as L
+import numpy as np
+from torch.utils.data import DataLoader
+from torch.utils.data import Subset
+from torchvision.transforms import AutoAugment
+from torchvision.transforms import Compose
+from torchvision.transforms import Normalize
+from torchvision.transforms import Resize
+from torchvision.transforms import ToTensor
+
+from probcal.custom_datasets import AAFDataset
+from probcal.custom_datasets import ImageDatasetWrapper
+
+
+class AAFDataModule(L.LightningDataModule):
+
+    IMG_SIZE = 224
+
+    def __init__(
+        self,
+        csv_file_path: str | Path,
+        root_dir: str | Path,
```

Comment:
It would be nice if this class took care of the downloading / CSV generation automatically. So in theory, all a new user would have to do is instantiate the AAFDataModule, call `prepare_data`, and everything would be set up. The COCO People dataset /datamodule is a good example of how this could look

Reply:
Okay! We will work on implementing this

Reply:
Fixed